src/io/c9/ace/Lib.js

```diff
@@ -0,0 +1,20 @@
+foam.CLASS({
```

Comment:
Why not use foam.LIB() instead? Because this is more lazy?

Reply:
I did it this way because foam.LIBs don't really store any state and I want to store this in a variable and a constant provides this functionality. But now that I think about it, I guess I could use foam.LIB and pass this function to memoize0? Maybe that's the better way to go since creating an instance of this "Lib" makes no sense.

Reply:
Ok now it's using foam.LIB and I put a fallback to a TextArea in case it fails to load. Let me know what you think.